lib.rs

```diff
@@ -142,15 +150,32 @@ pub type FnvHashMap<K, V> = HashMap<K, V, FnvBuildHasher>;
 #[cfg(feature = "std")]
 pub type FnvHashSet<T> = HashSet<T, FnvBuildHasher>;
 
-/// Const version of FNV hash.
-#[inline]
+/// Const version of FNV hash with standard initial state constant and prime factor.
+#[inline(always)]
 #[must_use]
 pub const fn fnv_hash(bytes: &[u8]) -> u64 {
-    let mut hash = INITIAL_STATE;
+    // SAFETY: This set of initial state and prime factor is already proven
+    unsafe { fnv_hash_with_state_and_prime(bytes, INITIAL_STATE, PRIME) }
+}
+
+/// Const version of FNV hash that the user can supply any initial state and prime.
+/// Keep in mind this is marked unsafe because we cannot prove that the user is really using a prime
```

Comment:
nit: must document that `bytes` being non-empty is a safety invariant

Reply:
you mean if bytes is empty, then it will return the initial state?

Reply:
No, this function is unsound if you pass it empty bytes. All callers should also be updated to explain how they uphold that invatiant